x-pack/plugins/infra/server/lib/adapters/nodes/metric_aggregation_creators/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { InfraMetricType } from '../../../../../common/graphql/types';
+import { count } from './count';
+import { cpu } from './cpu';
+import { load } from './load';
+import { memory } from './memory';
+import { rx } from './rx';
+import { tx } from './tx';
+
+export const metrics = {
```

Comment:
nitpicking: How about renaming this to `export const metricsAggregationCreators` too? Then it would match the directory name and the imported symbol would not have to be renamed.